LICENCE.txt

```diff
@@ -0,0 +1,190 @@
+EUROPEAN UNION PUBLIC LICENCE v. 1.2
```

Comment:
It is more future proof to point to the EUPL without version number, instead of including it:
See https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-547